### ISSUE TITLE
Linux installation problem: different python3 path in system console than in QGIS python console.

### ISSUE THREAD

**Issue by aloboa** [EnMAP-Box/enmap-box#833]
Step 4 of Option A/Install Python Dependencies 
(https://enmap-box.readthedocs.io/en/latest/usr_section/usr_installation.html#qgis-installation-on-linux-ubuntu)
indicates:
```
Open QGIS and the QGIS Python Console (Ctrl + Alt + P). Type the following and confirm with enter:
import sys; sys.executable
It shows the path of the python executable QGIS is using, usually /usr/bin/python3. It should be the same path as the command which python3 executed in the Terminal returns! Close QGIS.

```

but does not indicate what to do in case they do not match. In my case:
```
$ which python3
/home/alobo/miniconda3/bin/python3

```
while the python console within QGIS outputs:
```
import sys; sys.executable
'/usr/bin/python3'
```

**Comment by aloboa** [comment]
This mismatch occurs **only** if qgis is **not** started with:
"source ~/.virtualenvs/enmapbox/bin/activate && qgis %F"

**Comment by jakimowb** [comment]
@aloboa Look like you have different QGIS versions installed, 
one with conda/mamba, and one wit apt. 

Is the conda environment already activate when you open a new linux terminal?

Which QGIS binary is used when running `which qgis`?

**Comment by aloboa** [comment]
```
(base) alobo@pop-os:~$ which qgis
/usr/bin/qgis
(base) alobo@pop-os:~$ source ~/.virtualenvs/enmapbox/bin/activate
(enmapbox) (base) alobo@pop-os:~$ which qgis
/usr/bin/qgis
(enmapbox) (base) alobo@pop-os:~$
```

No problem with the python3 path  if I run qgis from within the environment.

**Comment by jakimowb** [comment]
@aloboa are the descriptions in https://enmap-box.readthedocs.io/en/latest/usr_section/usr_installation.html#qgis-installation-on-linux-ubuntu now fine for you?

**Comment by thielfab** [comment]
> @aloboa are the descriptions in https://enmap-box.readthedocs.io/en/latest/usr_section/usr_installation.html#qgis-installation-on-linux-ubuntu now fine for you?

Looks good to me, it is clearer now. 
@aloboa I think we already touched upon your environment issues in another issue, your `.bashrc` seems to be configured to load the conda environment automatically. https://github.com/EnMAP-Box/enmap-box/issues/403#issuecomment-1827586472

**Comment by aloboa** [comment]
Yes, right. I think the only remaining problem was that enpt_enmapboxapp could not be installed if the user had followed directions under
Option A: Install QGIS via the official repository[](https://enmap-box.readthedocs.io/en/latest/usr_section/usr_installation.html#option-a-install-qgis-via-the-official-repository)
AFAIK, this is solved (https://github.com/EnMAP-Box/enmap-box/issues/839), although I have not tested that myself (frankly, I now feel intimidated by EnMapBox installation problems and prefer to avoid "experiments").

**Comment by janzandr** [comment]
> Yes, right. I think the only remaining problem was that enpt_enmapboxapp could not be installed [...] AFAIK, this is solved (#839)

Correct
